app/Http/Controllers/Three/SignupsController.php

```diff
@@ -150,7 +149,7 @@ public function update(Request $request, Signup $signup)
      */
     public function destroy(Signup $signup)
     {
-        $signup->delete();
+        $this->signups->destroy($signup->id);
 
         return $this->respond('Signup deleted.', 200);
```

Comment:
Should this only be returned if `$this->signups->destroy($signup->id)` is `true`?

Reply:
I think so, just for safety, else return an error.

Reply:
@sbsmith86 I'm not sure what kind of exception to throw in this case, what do you think?

Reply:
we could just throw a `500` since if this method fails something weird is going on